packages/component/src/BasicToaster.js

```diff
@@ -0,0 +1,140 @@
+  return LEVEL_AS_NUMBER[level] || 5;
+}
+
+function compareLevel(x, y) {
+  return getLevelAsNumber(x) - getLevelAsNumber(y);
+}
+
+function sortNotifications(map) {
+  return (
+    Object.values(map)
+      .reduce((array, notification) => [...array, notification], [])
+      // We want the order reversed, most recent on top.
+      .sort(({ timestamp: x }, { timestamp: y }) => y - x)
+  );
+}
+
+const PASSTHRU_FN = value => value;
+
+const BasicToaster = () => {
+  const [{ toaster: toasterStyleSet }] = useStyleSet();
+  const [debouncedNotifications] = useDebouncedNotifications();
+  const [expanded, setExpanded] = useState(false);
+  const expandableElementId = useMemo(() => `webchat__toaster__list__${randomId()}`, []);
+  const headerElementId = useMemo(() => `webchat__toaster__header__${randomId()}`, []);
+  const renderToast = useRenderToast();
+  const toasterExpandText = useLocalize('TOASTER_HEADER_TEXT') || '';
+
+  const handleToggleExpand = useCallback(() => setExpanded(!expanded), [expanded, setExpanded]);
+  const sortedNotifications = useMemo(() => sortNotifications(debouncedNotifications), [debouncedNotifications]);
+  const sortedNotificationsWithChildren = useMemo(
+    () =>
+      sortedNotifications
+        .map(notification => {
+          const children = renderToast({ notification });
+
+          return children && { children, notification };
+        })
+        .filter(PASSTHRU_FN),
+    [renderToast, sortedNotifications]
+  );
+
+  const expandable = sortedNotificationsWithChildren.length > 1;
+  const [highestLevel] = sortedNotificationsWithChildren.map(({ notification: { level } }) => level).sort(compareLevel);
+
+  useEffect(() => {
+    !expandable && setExpanded(false);
+  }, [expandable]);
+
+  return (
+    <div
+      aria-labelledby={headerElementId}
+      aria-live="polite"
+      aria-relevant="additions text"
+      className={classNames(ROOT_CSS + '', toasterStyleSet + '', 'webchat__toaster', {
+        'webchat__toaster--expandable': expandable,
+        'webchat__toaster--expanded': expanded,
+        'webchat__toaster--error': highestLevel === 'error',
+        'webchat__toaster--info': highestLevel === 'info',
+        'webchat__toaster--success': highestLevel === 'success',
+        'webchat__toaster--warn': highestLevel === 'warn'
+      })}
+      role="log"
+    >
+      {expandable && (
+        <button
+          aria-controls={expandableElementId}
+          aria-expanded={expanded}
+          className="webchat__toaster__header"
+          id={headerElementId}
+          onClick={handleToggleExpand}
+          type="button"
+        >
+          <div aria-hidden={true} className="webchat__toaster__expandLevelIconBox">
+            <NotificationIcon className="webchat__toaster__expandLevelIcon" level={highestLevel} />
+          </div>
+          <div className="webchat__toaster__expandText">
+            {toasterExpandText.replace('$1', sortedNotificationsWithChildren.length)}
```

Comment:
Should `useLocalize` handle updating the number of notifications?

Reply:
Let's update it my another PR, which will redo those strings.